primitives/state-machine/src/testing.rs

```diff
@@ -320,4 +339,20 @@ mod tests {
 			assert!(ext.child_storage(&child_info, &b"dog2"[..]).is_some());
 		}
 	}
+
+	#[test]
+	fn as_backend_generates_same_backend_as_commit_all() {
+		let mut ext = TestExternalities::<BlakeTwo256, u64>::default();
+		{
+			let mut ext = ext.ext();
+			ext.set_storage(b"doe".to_vec(), b"reindeer".to_vec());
+			ext.set_storage(b"dog".to_vec(), b"puppy".to_vec());
+			ext.set_storage(b"dogglesworth".to_vec(), b"cat".to_vec());
+		}
+
+		let backend = ext.as_backend();
```

Comment:
Do we still need this function? Doesn't `commit_all` accomplish the same now?

Reply:
The `PartialEq` function still uses this.

Reply:
Otherwise this function would have been gone with the opening of the pr ;)

Reply:
Ahh okay. But then it could also be private, right?

Reply:
Good point :P Done